setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup
 
-VERSION = '0.4.11'
+VERSION = '0.5.0'
```

Comment:
Yes, please. For this repo we prefer to make the changes to setup.py, CHANGELOG, and docs/conf.py [all in a discreet PR like this example one](https://github.com/edx/bok-choy/pull/119). Mostly because otherwise at least one of them ends up being forgotten. :|

Let's get this PR merged in first, and then @benpatterson or I will release the new version, which is represented by TE-1218 and in our current sprint.